hair_salon/data/ir_model_fields.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="x_break_end_appointment_model" model="ir.model.fields">
```

Comment:
We should maybe make sure that those float are reset to 0 if the user unchecks x_break. What do you think?
For that, I'd  move the field to a compute (not readonly and stored), with the python dealing with this only case.